notes/meeting11.md

```diff
@@ -0,0 +1,253 @@
+- obviously advanced, not in MVP, but end product would have it.
+- [hester] anyone else see relevance of arbitration anywhere else
+- [julien] [[Corey didn't hear this, reading email]]
+- [ricardo] you can't take an action to change something in Sticker market
+- maybe we look at kleros more, and set DAO to do arbitration
+- [hester] if this comes up anywhere, it seems most reasonable to ask what Teller Network has learned as they've been looking most into it so far
+- [oskar] other staking mechanisms could be looked into and be considered
+- [ricardo] we should start implementing these report features within the app so at least we can have something.  Have expectation of what we need to solve.  If it doesn't work perfectly, it shows what we need to fix instead of having nothing
+- swarm for opening arbitration cases and
+- [oskar] common law type interface, start by underspecifying it and evolve from there, maybe
+
+### CCO Re-license
+- [jarrad] basically, depending on regulatory climates, distribution policies that we have to adhere to, it seems gmbh may be forced to create binaries in conflict of what our vision is about.
+- that means we have to use our codebase into public domain, which is difficult actually
+- if we relicense to CC0 it waives all things, but also makes it [[Corey didn't get this part]]
+- so we put into public domain, and maintain a fork of it
+- This requires **anyone who has contributed to the codebase** might have to sign some paperwork and get their permission to do this.
+- [julien] does this have an impact on dependencies?
+- [jarrad] not really, they're all copyleft in some form, and CC0 is compatible with that, public domain more so.
+- [jarrad] its best if we attempt reach out to everyone, even if they're dead
+- [pedro] I thought we already gave up the right to our work in contracts
+    - only works for contractors
+- [igor] do we need a dependency audit for this?
+- [jarrad] naw, don't think so.
+- [oskar] what would you like to see to get this done? how do you practially look at doing this
+- [jarrad] make everyone aware of this, gather everyone's contact details and emails, then email them with situation and paperwork, and collect signatures as best as possible.
+- [oskar] bounties contributors may mean situations.
+    - we might have to redo work of these folks
+- this is where decoupling protocol from implementation would be useful.
+- [jarrad] reference implementation will be new codebase anyway (probably)
+
+### Blocking PRs with automated tests
+- [pedro] period of time during pipeline of PR where we weren't checking if automated tests had been run
+- I fixed last week that a PR starts in a failed state, and automated tests changes that, which helps a lot
+- We need to change some things in the way the repo is set up.  Tried to do this with Jakub last week but we had issues, saw that we need to investigate further to step on other's shoes.
```

Comment:
```suggestion
- We need to change some things in the way the repo is set up.  Tried to do this with Jakub last week but we had issues, saw that we need to investigate further to avoid stepping on other's shoes.
```

Reply:
fixed